ff/Cargo.toml

```diff
@@ -16,12 +16,12 @@ edition = "2021"
 [dependencies]
 bitvec = { version = "1", default-features = false, optional = true }
 byteorder = { version = "1", default-features = false, optional = true }
-ff_derive = { version = "0.12.1", path = "ff_derive", optional = true }
-rand_core = { version = "0.6", default-features = false }
-subtle = { version = "2.2.1", default-features = false, features = ["i128"] }
+ff_derive = { version = "0", path = "ff_derive", optional = true }
```

Comment:
seems not a good idea.   v0.x and v0.y don't need to be compatible in sem ver

```ff_derive = { version = "0",```

Reply:
They actually need to be compatible **in semantic versioning**.  But you are right that **in practice** they often aren't.

Fixed.